src/fetch.rs

```diff
@@ -0,0 +1,54 @@
+use crate::command::Command;
+use async_trait::async_trait;
+use serde::de::DeserializeOwned;
+use wasm_bindgen::{prelude::*, JsCast};
+use wasm_bindgen_futures::JsFuture;
+use web_sys::{Request, RequestInit, RequestMode, Response};
+
+pub struct Fetch<Data, Message> {
```

Comment:
This is a basic networking command to make GET requests and deserialize the results.